go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/go-ini/ini v1.46.0
 	github.com/google/go-cmp v0.5.5
 	github.com/mitchellh/copystructure v1.0.0
+	github.com/nsf/jsondiff v0.0.0-20210303162244-6ea32392771e
```

Comment:
Is this used now? maybe you can revert the changes to `go.mod` and `go.sum` now.

Reply:
Package has been removed.